src/collectors/windows.plugin/perflib-processor.c

```diff
@@ -3,6 +3,10 @@
 #include "windows_plugin.h"
 #include "windows-internals.h"
 
+#define _COMMON_PLUGIN_NAME "freebsd.plugin"
+#define _COMMON_PLUGIN_MODULE_NAME "freebsd"

```

Comment:
Why are these `define`s freebsd in `perflib-processor.c`?

Reply:
Thanks!